wandb/data_types.py

```diff
@@ -31,40 +31,46 @@
 import six
```

Comment:
move this file into the wandb/sdk/data_type folder... and break it into modules as well...
you could configure `mypy.ini` to ignore type checks for specific files. see example here: https://github.com/wandb/client/blob/master/mypy.ini#L110

Reply:
I totally agree (: I think I'd like to do this in a followup PR. Trying to not change too much in terms of namespace and files in one move.